### ISSUE TITLE
TokenGS performance and finetuning on synthetic multi-view data

### ISSUE THREAD

**Issue by hlwdy** [nv-tlabs/TokenGS#12]
Hi, thanks for releasing TokenGS!

We are currently exploring using TokenGS as a backbone for downstream tasks on synthetic multi-view data. Before building on top of the TokenGS representation, we would like to make sure that we can obtain reasonably good static reconstruction quality on synthetic scenes.

We first tested the released `dl3dv_latent_6v_lpips` checkpoint on [Syn4D](https://huggingface.co/datasets/Syn4D/Syn4D). The reconstruction usually captures the coarse scene structure, but the fine geometry and appearance are noticeably worse than what we observe on DL3DV scenes.

We tried to match the TokenGS camera and image preprocessing as closely as possible:

* converting the provided camera poses to the OpenCV camera convention expected by TokenGS;
* constructing the intrinsics from the provided camera parameters and updating them consistently after cropping/resizing;
* applying the center-crop/resize preprocessing;
* applying the same camera normalization used by the corresponding TokenGS preset;

For the released checkpoint, we found that using a fixed scene rescaling factor around `0.15` gives better results. However, even after this adjustment, the synthetic reconstruction quality is still noticeably worse than on DL3DV.

without TT refine, 6 input views:

<img width="2688" height="852" alt="Image" src="https://github.com/user-attachments/assets/13a13272-48fe-4960-99f8-e6dc6cf399c4" />

We observe that the reconstruction quality on Syn4D is already noticeably degraded even when rendering the input views.

Since the repository also contains the Kubric training setup, we would like to better understand the expected behavior of TokenGS on synthetic data:

1. How well does the static TokenGS model reconstruct your Kubric data?

2. Where can we download the exactly same Kubric/Kubric4D dataset used for the released experiments?

3. We noticed the `finetune_dl3dv_kubric_static` preset in `options.py`. Was this static stage used when training the released Kubric dynamic model, or is the released dynamic checkpoint trained directly from the DL3DV base checkpoint using `finetune_dl3dv_kubric_dyn_release`?

4. If we want to adapt TokenGS to another synthetic dataset such as Syn4D for use as a downstream backbone, what finetuning strategy would you recommend?
   For example:
   * starting from `dl3dv_latent_6v_lpips`;
   * starting from the DL3DV base checkpoint;
   * following `finetune_dl3dv_kubric_static` first?

A reference Kubric scene/subset together with the expected static reconstruction quality would be especially useful for us, since it would help distinguish a remaining camera/preprocessing issue from a synthetic-to-real domain gap.

Thanks!
